provider/datagen/src/transform/cldr/currency/mod.rs

```diff
@@ -0,0 +1,357 @@
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
```

Comment:
issue: this needs to be feature-gated

Reply:
I have made `mod currency` feature gated.